src/js/view/view.js

```diff
@@ -149,6 +154,11 @@ function View(_api, _model) {
 
         _resizeMedia(containerWidth, containerHeight);
         _captionsRenderer.resize();
+
+
+        if (_floatingConfig) {
```

Comment:
now that the throttled handler checks for mobile and orientation, no need to do it here